packages/paste-website/src/pages/primitives/listbox-primitive/index.mdx

```diff
@@ -0,0 +1,322 @@
+The `onSelect` event handler will fire whenever the item is clicked, or the spacebar or enter key is pressed. Use this to respond to your users' selection interactions.
+
+### Horizontal Listbox
+
+Our listbox is vertical by default. To change it to horizontal, specify the orientation when initializing the state with `useListboxPrimitiveState({orientation: 'horizontal'})`:
+
+<LivePreview
+  scope={{
+    useListboxPrimitiveState,
+    ListboxPrimitive,
+    ListboxPrimitiveItem,
+  }}
+  noInline
+  showOverflow
+>
+  {horizontalExample}
+</LivePreview>
+
+### Listbox with groups
+
+Use `ListboxPrimitiveGroup` to create different groupings within the same listbox:
+
+<LivePreview
+  scope={{
+    useListboxPrimitiveState,
+    ListboxPrimitive,
+    ListboxPrimitiveGroup,
+    ListboxPrimitiveItem,
+    Stack,
+  }}
+  noInline
+  showOverflow
+>
+  {groupsExample}
+</LivePreview>
+
+### Multiselect Listbox
+
+Our listbox is set for single selection by default. To change it to allow multiple selections, specify the variant as `multiple` on `ListboxPrimitive`:
+
+<LivePreview
+  scope={{
+    useListboxPrimitiveState,
+    ListboxPrimitive,
+    ListboxPrimitiveItem,
+  }}
+  noInline
+  showOverflow
+>
+  {multipleExample}
+</LivePreview>
+
+### Dual Listboxes
+
+Listboxes can be used in any situation where a user needs to select items in a list or rearrange a list. One example is a UI where users need to move items between two lists like this:
+
+<LivePreview
+  scope={{
+    useListboxPrimitiveState,
+    ListboxPrimitive,
+    ListboxPrimitiveItem,
+    Box,
+    Button,
+    CheckboxCheckIcon,
+    PlusIcon,
+    MinusIcon,
+    Column,
+    Grid,
+    Stack,
+  }}
+  noInline
+  showOverflow
+>
+  {dualExample}
+</LivePreview>
+
+### Custom styling
+
+You can provide custom styling to the primitive listbox by utilizing the `as` prop on each component.
+
+The listbox primitive does not come with any styling, and thus you could mix the functionality of it with another component by using the _as_ prop. By doing so, you can get styling from another component, and listbox functionality from this primitive.
+
+Because these are not styled, rendering any of them _as_ another component you can mix the functionality of two components together. Styling from one, listbox functionlity from the primitive component.
+
+In the example below, we import the Paste Button `import {Button} from '@twilio-paste/button';` and use it as the Listbox items via the `as` prop.
+
+<LivePreview
+  scope={{
+    useListboxPrimitiveState,
+    ListboxPrimitive,
+    ListboxPrimitiveItem,
+    Button,
+    Stack,
+    CheckboxCheckIcon,
+  }}
+  noInline
+  showOverflow
+>
+  {customExample}
+</LivePreview>
+
+---
+
+## Usage Guide
+
+This package is a wrapper around the [Reakit Composite](https://reakit.io/docs/composite/). If you’re wondering why we wrapped that package into our own, we reasoned that it would be best for our consumers’ developer experience. For example:
```

Comment:
Maybe this could be shortened up a bit:

"This package is a wrapper around the Reakit Composite, as we believe it's best for our consumers' developer experience. For example:"